std/experimental/allocator/building_blocks/region.d

```diff
@@ -56,14 +56,12 @@ struct Region(ParentAllocator = NullAllocator,
 
     private void* _current, _begin, _end;
 
-    pure nothrow @trusted @nogc
-    private void* roundedBegin() const
+    private void* roundedBegin() const pure nothrow @trusted @nogc
```

Comment:
I'm not sure if this makes it easier to read the function attributes or not.
As @andralex said, we need to come up with a consistent way of writing those;
there are currently a few styles in phobos.

Reply:
I'm leaving it as is, then.

Reply:
We already had this discussion before and thus even put it in the [DStyle](https://dlang.org/dstyle.html#phobos_attributes)

> Attributes should be listed in alphabetical ordering, e.g. const @nogc nothrow pure @safe (the ordering should ignore the leading @).

Though without automation no one really felt the need to change things and adapt his/her own style to it.